x-pack/plugins/security_solution/cypress/e2e/urls/not_found.cy.ts

```diff
@@ -30,7 +30,8 @@ describe('Display not found page', () => {
     visit(TIMELINES_URL);
   });
 
-  it('navigates to the alerts page with incorrect link', () => {
+  // TODO: We need to determine what we want the behavior to be here
```

Comment:
The current alert path is `/alerts/:id` and this currently shows an error if the `:id` does not return data